openmp/libompd/gdb-plugin/ompdModule.c

```diff
@@ -1180,7 +1180,7 @@ static PyObject *call_ompd_get_icv_from_scope(PyObject *self, PyObject *args) {
 
   if (retVal != ompd_rc_ok) {
     if (retVal != ompd_rc_incomplete) {
-      _printf("An error occurred when calling ompd_get_icv_from_scope(%i, %i): "
+      _printf("An error occurred when calling ompd_get_icv_from_scope(%i, %" PRIu64 "): "
               "Error code: %d",
               scope, icvId, retVal);
```

Comment:
```suggestion
      _printf(
              "An error occurred when calling ompd_get_icv_from_scope(%i, %" PRIu64 
              "): Error code: %d", scope, icvId, retVal);
```
This should make clang-format happier

Reply:
Thanks